### PR TITLE
[Reference cleanup]LPS-162275

### PR DESCRIPTION
**Pull request by JanisZhang**
Hi @dantewang @tinatian,

Resend, fix unit test.

This PR is to convert method references to field references in dynamic-data-mapping (Misc).

https://issues.liferay.com/browse/LPS-162275

Previous PR:https://github.com/liferay-core-infra/liferay-portal/pull/1557

### CODE REVIEW

File: modules/apps/dynamic-data-mapping/dynamic-data-mapping-data-provider-impl/src/main/java/com/liferay/dynamic/data/mapping/data/provider/internal/rest/DDMRESTDataProvider.java [hunk 1]
```diff
@@ -641,6 +633,7 @@ private void _setMultiVMPool(MultiVMPool multiVMPool) {
 	@Reference
 	private JSONWebServiceClientFactory _jsonWebServiceClientFactory;
 
+	@Reference
```
Comment (dantewang) [inline modules/apps/dynamic-data-mapping/dynamic-data-mapping-data-provider-impl/src/main/java/com/liferay/dynamic/data/mapping/data/provider/internal/rest/DDMRESTDataProvider.java:636]: @JanisZhang You can't reference a portal cache. You should reference `MultiVMPool`, and create the portal cache in activate, and remove it in deactivate.

**Review by dantewang — CHANGES_REQUESTED** [review]

File: modules/apps/dynamic-data-mapping/dynamic-data-mapping-data-provider-impl/src/main/java/com/liferay/dynamic/data/mapping/data/provider/internal/rest/DDMRESTDataProvider.java (hunk 1 above)
Comment (JanisZhang) [reply modules/apps/dynamic-data-mapping/dynamic-data-mapping-data-provider-impl/src/main/java/com/liferay/dynamic/data/mapping/data/provider/internal/rest/DDMRESTDataProvider.java:636]: @dantewang  Noted, Thanks.

**Comment by JanisZhang** [comment]
Hi @dantewang, force-pushed.
Hi @catarineaguiar ,Could you please help to scan the test results? Thanks.

**Comment by dantewang** [comment]
ci:test:relevant

**Review by dantewang — APPROVED** [review]

**Comment by catarineaguiar** [comment]
@JanisZhang no regressions found.

**Comment by tinatian** [comment]
FYI @shuyangzhou See https://github.com/brianchandotcom/liferay-portal/pull/122729

**Pull request closed without merging**